openstudiocore/src/model/AirLoopHVACOutdoorAirSystem.cpp

```diff
@@ -90,6 +92,8 @@ namespace detail {
   {
     std::vector<ModelObject> result;
     result.push_back(this->getControllerOutdoorAir());
+    std::vector<AirflowNetworkDistributionNode> myAFNItems = getObject<ModelObject>().getModelObjectSources<AirflowNetworkDistributionNode>(AirflowNetworkDistributionNode::iddObjectType());
```

Comment:
If I am understanding correctly, a distribution node references a component.  A distribution linkage references two nodes and a component between them.  So an AirLoopHVACOutdoorAirSystem have multiple distribution node children?  Would these nodes be forced to the same air conditions?  An AirLoopHVACOutdoorAirSystem cannot have a distribution linkage as a child?